GlimpseImputationPipeline/Glimpse2MergeBatches.wdl

```diff
@@ -5,42 +5,68 @@ workflow Glimpse2MergeBatches {
         Array[File] imputed_vcfs
         Array[File] imputed_vcf_indices
 
+        Array[File?]? qc_metrics
```

Comment:
is it really necessary for this to be `Array[File?]?` as opposed to just `Array[File]?` 

I get that it's slightly more flexible, but the use case of having qc_metrics for some but not all of the batches doesn't seem particularly useful.  And honestly, would we want to allow that in the first place?  or does it make more sense to say that if your passing in metrics files to merge, you should have them for every batch?

Reply:
Yes, I agree that metrics should not be passed for some and not all batches. The reason for the `File?` is that if `collect_qc_metrics` is set to false then Glimpse2Imputation returns null as the `qc_metrics` output, which means that Glimpse2MergeBatches actually receives `[null, null]` in the case of two batches. That's why I implemented it this way to keep it as flexible as possible, but I'm debating to implement it differently, by making it an `Array[File]` and calling it either with or without that argument from Glimpse2ImputationInBatches